functions/streamtoidv/utils.py

```diff
@@ -0,0 +1,28 @@
+__version__ = '0.0.1'
+
+import os
+import logging
+import time
+from datetime import datetime
+from logging import NullHandler
+
+
+def set_stream_logger(name="cis-validator", level=logging.INFO, format_string=None):
+    """
+    :Stream logger class borrowed from https://github.com/threatresponse/aws_ir
```

Comment:
you can either copy the file and specify which files/code are affected by it, or copy paste the license file contents at the top of the file or function that contains the licensed code

note: this depends on the license, but in this case its MIT so both ways would work - but linking does not. (some other licenses allow linking)

note2: a lot of projects dont do due diligence here and whoever pickup the code also infringe the license - which nobody cares about unless its put into proprietary software. that said, for attribution reasons and ethics, in addition to be an actual legal requirement i'm always trying to be careful with licenses